### PR TITLE
ADD: Kimi no na wa ost and full metal panic opening

### PR DESCRIPTION
**Pull request by SoldierCorp**
More songs for this issue #195 

#### This PR is for [Kimi no na wa, Full metal panic].

- [ ] This issue is for enhancement/bug fixing.
- [x] This issue is an addition to JSON database.
	- [ ] Pull Request Title is of form `Added [Kimi no na wa, Full metal panic]`.
- [ ] This addition is already on ongaku.
	- [ ] This PR is an improvement to current addition.
- [ ] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/anshumanv/ongaku/blob/master/CONTRIBUTING.md).
- [ ] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

File: js/data.json [hunk 1]
```diff
@@ -1262,6 +1268,18 @@ osts = [
 		"link" : "https://my.mixtape.moe/swjtom.mp3",
 		"name" : "Boku no Hero Academia S2 OST-2 Jet Set Run",
 		"img" : "https://images6.alphacoders.com/820/820024.png"
+	},
+
+	{
+		"link" : "https://s1.vocaroo.com/media/download_temp/Vocaroo_s1b0f75sq5EN.mp3",
+		"name" : "Kimi no na wa OST - Yumetourou Radwimps",
+		"img" : "https://www.koi-nya.net/img/subidos_posts/2017/01/Kimi-no-na-wa-protagonistas.jpg"
```
Comment (anshumanv) [inline js/data.json]: Can you use some other img link for this? this link is failing the Travis build for some reason

**Comment by anshumanv** [comment]
Hey, @SoldierCorp thanks for the PR. 😄 
Please check comments

**Review by anshumanv — CHANGES_REQUESTED** [review]

File: js/data.json [hunk 2]
```diff
@@ -1262,6 +1268,18 @@ osts = [
...
+	},
+
+	{
+		"link" : "https://s1.vocaroo.com/media/download_temp/Vocaroo_s1oMIbR1BFgl.mp3",
+		"name" : "Kimi no na wa OST - Zen zen zense Radwimps",
+		"img" : "https://www.koi-nya.net/img/subidos_posts/2017/01/Kimi-no-na-wa-protagonistas.jpg"
```
Comment (anshumanv) [inline js/data.json]: Please use some other image for this one. Redundant images aren't very appealing.

**Comment by SoldierCorp** [comment]
Sure,

So the issue is to use another image for the `zen zen` song and use a different link for `Yumetourou` one?

**Comment by anshumanv** [comment]
Yea use different images for different tracks.

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Amazing! 👍

**Comment by anshumanv** [comment]
Please add more tracks if you wish. 😄

**Comment by SoldierCorp** [comment]
Absolutely :)

**Pull request merged**